flamingpy/utils/viz.py

```diff
@@ -284,6 +285,138 @@ def draw_EGraph(
 
     return fig, ax
 
+@mpl.rc_context(plot_params)
```

Comment:
I am not sure if this has any effect on the resulting plotly plot since this is a matplotlib decorator. Have you checked this?

Reply:
It won't...